app/views/groups/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="containers">
+  <h5 class='text-center display-6'>TRANSACTION</h5>
+  <div class="p-4 pb-0">
+    <%=image_tag @group.image, class: 'img-thumbnail ratio ratio-21x9' %>
+    <p class="text-muted text-end pt-2"><%= @group.name %></p>
+  </div>
+</div>
+<div class="container rounded">
+  <div class="container p-4 pt-0 d-flex justify-content-between">
+		<h5 class="fw-bold">Total Amount:</h5>
+    <div class="d-flex amount-details">
+		<p class="fw-bold">$<%= @group.total_amount.round(2) %></p>
+  </div>
```

Comment:
- [ ] Kindly ensure the total amount for the group is displayed at the top of the page (above the image.).

Reply:
done